tools/mo/openvino/tools/mo/front/common/partial_infer/utils.py

```diff
@@ -303,3 +303,12 @@ def is_dynamic_slice(s: [slice, int, None]):
     return isinstance(s, slice) and (s.start is dynamic_dimension or
                                      s.stop is dynamic_dimension or
                                      s.step is dynamic_dimension)
+
+
+def reverse_bypass_infer(node, ports: List[int] = None):
```

Comment:
What `ports` function argument is needed for?

Reply:
`ports` argument is used to specify manually for which inputs ranks/partial_shape should be deduced. 

E.g. if there is a `Split` op, it has 2 inputs and ranks/partial_shapes should be deduced only `in_port = 0`. `in_port = 1` contains only `num_splits` and should not be changed
https://github.com/openvinotoolkit/openvino/blob/f8136c442b9e5ace226ff531d6726551b80b3b09/tools/mo/openvino/tools/mo/ops/split.py#L232